.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "retail",
+    "name_pretty": "Retail",
```

Comment:
Should this be `Recommendations AI`, perhaps we can bounce this by the engineer you've been talking with about this product.